python/sdk/prelude_sdk/controllers/iam_controller.py

```diff
@@ -10,10 +10,14 @@ def __init__(self, account):
         self.account = account
 
     @verify_credentials
-    def new_account(self, user_email: str, user_name: str):
+    def new_account(self, user_email: str, user_name: str, company: str = None):
+        body = dict(handle=user_email, user_name=user_name)
```

Comment:
user_name and company are both optional - it'd be nice to be consistent (i'd go with giving user name an `if user_name:` clause like you do with company)